tests/cdiconfig_test.go

```diff
@@ -348,6 +370,15 @@ var _ = Describe("CDI ingress config tests", func() {
 			defaultUrl = *config.Status.UploadProxyURL
 		}
 		By("Making sure no url is set")
+		config, err := f.CdiClient.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
```

Comment:
Changed setup & teardown around this test suite to address failures in lanes
(We don't take into account that env might have actual override set)